packages/core/src/__tests__/ceramic-feed.test.ts

```diff
@@ -0,0 +1,158 @@
+import { expect, describe, test, beforeEach, afterEach } from '@jest/globals'
+import { TestUtils, type StreamState, type IpfsApi } from '@ceramicnetwork/common'
+import { TileDocument } from '@ceramicnetwork/stream-tile'
+import { createIPFS, swarmConnect, withFleet } from '@ceramicnetwork/ipfs-daemon'
+import type { Ceramic } from '../ceramic.js'
+import { createCeramic as vanillaCreateCeramic } from './create-ceramic.js'
+import { Model, ModelDefinition } from '@ceramicnetwork/stream-model'
+import { take } from 'rxjs'
+import { Document } from '../feed.js'
+
+function createCeramic(
+  ipfs: IpfsApi,
+  anchorOnRequest = false,
+  streamCacheLimit = 100
+): Promise<Ceramic> {
+  return vanillaCreateCeramic(ipfs, {
+    anchorOnRequest,
+    streamCacheLimit,
+  })
+}
+
+describe('Ceramic feed', () => {
+  let ipfs: IpfsApi
+  let ceramic: Ceramic
+  beforeEach(async () => {
+    ipfs = await createIPFS()
+    ceramic = await createCeramic(ipfs)
+  })
+
+  afterEach(async () => {
+    await ceramic.close()
+    await ipfs.stop()
+  })
+
+  test('add entry after creating/updating stream', async () => {
+    const feed: Document[] = []
+    const s = ceramic.feed.aggregation.documents.subscribe((s) => {
+      feed.push(s)
+    })
+    const original = `world-${Math.random()}`
+    const tile = await TileDocument.create(ceramic, { hello: original }, null, {
+      anchor: false,
+    })
+    const updated = `world-1-${Math.random()}`
+    await tile.update({ hello: updated }, undefined, { anchor: false })
+    s.unsubscribe()
+    // One entry after create, and another after update
+    expect(feed.length).toEqual(2)
+
+    expect(feed[0].commitID).not.toBe(feed[1].commitID)
+    expect(feed[0].metadata).toStrictEqual(feed[1].metadata)
+    expect(feed[0].content).toStrictEqual({ hello: original })
+    expect(feed[1].content).toStrictEqual({ hello: updated })
+  })
+
+  test('add entry after loading pinned stream/pubsub ', async () => {
+    await withFleet(2, async ([ipfs1, ipfs2]) => {
+      await swarmConnect(ipfs2, ipfs1)
+
+      const ceramic1 = await createCeramic(ipfs1)
+      const ceramic2 = await createCeramic(ipfs2)
+      const content = { test: 1223 }
+      const updatedContent = { test: 1335 }
+      const feed1: Document[] = []
+      const feed2: Document[] = []
+      const s1 = await ceramic1.feed.aggregation.documents.subscribe((s) => {
+        feed1.push(s)
+      })
+
+      const s2 = await ceramic2.feed.aggregation.documents.pipe(take(3)).subscribe((s) => {
+        feed2.push(s)
+      })
+      const stream1 = await TileDocument.create(ceramic1, content, null, {
+        anchor: false,
+        publish: true,
+      })
+      // load stream on ceramic node 2
+      await TileDocument.load(ceramic2, stream1.id)
+
+      await stream1.update(updatedContent, null, { publish: true })
+      await TestUtils.delay(500)
+
+      expect(feed1.length).toEqual(3) // create + anchor + update
```

Comment:
why is there an anchor since the stream is created with `anchor:false`?

Reply:
After the update the anchor status changes from 0 to 1

Reply:
aha, that exposes an interesting property of this feed - there's an event emitted every time the anchor status changes, even if there is no corresponding commit or change to the actual log/data of the stream.  This kind of relates to my comment elsewhere about whether or not we want anchorStatus to show up in the feed at all.  That's really a question for product though.

I suspect if you set `anchor:false` in the args for the update then the feed length will go to 2 here.

Reply:
I see, I guess we'll revisit this when we figure out if needed or not first.

Reply:
@JulissaDantes We should add tests for both anchor:true and anchor:false, just to record behavior.

Reply:
I included the `anchor:false` in the update to test behavior without anchor, and if it gets an event when anchoring is true gets tested at the `add entry after anchoring stream` test.